### ISSUE TITLE
Chromecast & transcoding

### ISSUE THREAD

**Issue by NessunoIM** [koying/SPMC#300]
It is planning to add support to Chromecast & transcoding?

**Comment by heymoe** [comment]
You're likely better off looking at Plex for Chromecast & transcoding support.

**Comment by Aghabeiki** [comment]
But why?

**Comment by heymoe** [comment]
Plex and SPMC are both based off of Kodi/XBMC. Plex's first public release was back in 2010, 2 years after the first code commit happened that was the start of Plex. It then took the Plex team almost another 7 years to release version 1.0 of Plex and they continue to improve it.

I don't know how many people work at Plex but I'm sure it's more than 1 person and they took over 7 years working on Plex, which is basically Kodi/XBMC/SPMC with a server component that can transcode and cast.

I doubt Koying / SPMC will want to take the time and effort to re-invent the wheel that Plex has already created and can be used for free.

**Comment by Aghabeiki** [comment]
yes it is, but Kodi is Kodi and Plex is Plex, I work with both of them and I prefer Kodi at all, any why, I agree totally with you but about re-invent the wheel, if we think like this, so right now should not be Linux here! because before it Unix and ... Exist, so let's talk about how we can develop this thing for us, 

anyone has Idea about it?

**Comment by heymoe** [comment]
I get the argument against not doing something just because someone has already done it. It's true if others don't re-invent the wheel we missing out on new innovations, features, etc.. 

My main point is that unless more development resources become available to SPMC, Koying by himself is unlikely to take on building a new transcoding and casting system into SPMC seeing that it's taken Plex several years to get their system to where it is now with a team of developers behind it.

Also there are other applications that are competing more with Plex such as Emby which does transcoding and casting and also but provides more customization features that Plex does not have from my understanding. I personally have not tried it though. 

I personally use both Kodi/SPMC and Plex in my environment. Kodi/SPMC is my main interface on my big screen. I like the customization that I can do with the Interface and seems to be updated faster for new audio and video formats. I use Plex on my portable devices (phones, tablets) since it's quicker and easier to just install the Plex Player app and have access to my content then installing Kodi/SPMC to a portable device and customizing it to access you content plus I use Plex when I'm remote / traveling to access my media on the go which isn't as easy to do with Kodi/SPMC but the whole transcoding ability comes in handy for remote access. . 

Plex also released an official Kodi app that lets you access your Plex library from within Kodi so if you like Kodi/SPMC but want to access Plex for content you can now. There have been other apps that do the same thing but this is an official one from Plex. You do need a Plex Pass right not to access it since it's still in beta.
